packaging/conda/build.sh

```diff
@@ -15,7 +15,8 @@ fi
 cmake --fresh --preset $CMAKE_PRESET -DBUILD_JARS=OFF -DTESTING=OFF
 
 # Build MODL and MODL_Coclustering
-cmake --build --preset $CMAKE_PRESET --parallel --target MODL MODL_Coclustering KhiopsNativeInterface KNITransfer 
+cmake --build --preset $CMAKE_PRESET --parallel \
+  --target MODL MODL_Coclustering KhiopsNativeInterface KNITransfer
```

Comment:
This change does not seem related to the current PR; was it required by the pre-commit hooks?

Reply:
No, just a small editorial change because the line was too long.

Reply:
Ok.